akvo/akvo-rsr
# [#4955] password history

# TODO / Done

Summarize what has been changed / what has to be done in order to finalize the PR.

 - [x] Implement Password history log
 - [x] Implement Reuse limit rule
 - [x] Implement Forced password changes

# Test plan

What tests are necessary to ensure this works or doesn't break anything working

 
## Manual testing

- Create password policy config on your main organisation, set the password to expire in 1 day
- Change your password to enable the password  history log for your account
- Logout and wait until your password expired (1 day)
- Login then you should see the change password form after logged in.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205121015665343
